frontend/src/lib/components/Cards/InsightCard/InsightDetails.tsx

```diff
@@ -289,15 +288,20 @@ export function FiltersSummary({ filters }: { filters: Partial<FilterType> }): J
     )
 }
 
-export function BreakdownSummary({ filters }: { filters: Partial<FilterType> }): JSX.Element {
+export function BreakdownSummary({ filters }: { filters: Partial<FilterType> }): JSX.Element | null {
+    if (filters.breakdown_type == null || filters.breakdown == null) {
+        return null
+    }
+
+    const breakdownArray = Array.isArray(filters.breakdown) ? filters.breakdown : [filters.breakdown]
+
     return (
         <>
             <h5>Breakdown by</h5>
             <section>
-                <TaxonomicBreakdownFilter
-                    isTrends={filters.insight === InsightType.TRENDS}
-                    breakdownFilter={filters as BreakdownFilter}
-                />
+                {breakdownArray.map((breakdown) => (
+                    <BreakdownTag key={breakdown} breakdown={breakdown} breakdownType={filters.breakdown_type} />
```

Comment:
We use a slightly smaller font size in `InsightDetails` (0.8125 = 13px, instead of 14px). The breakdown pills should follow that too, currently they seem a bit inflated.

Reply:
BTW while we're here can you add `vertical-align: -0.3em` to `.SeriesDisplay__raw-name`? Just noticed they look like this, which is not right: 😅 
<img width="363" alt="Screenshot 2023-08-24 at 20 01 53" src="https://github.com/PostHog/posthog/assets/4550621/c2b3590c-4bbf-4d36-8435-dc7fa7631f76">

Reply:
Done and done :)

**Before**
<img width="1095" alt="after" src="https://github.com/PostHog/posthog/assets/1851359/2cfd595e-f689-40f0-b675-22efa9c9ce4e">

**After**
<img width="1099" alt="before" src="https://github.com/PostHog/posthog/assets/1851359/b2cdb286-abb0-4efa-b3de-e2ae6cc0d8f0">